app/build.gradle

```diff
@@ -20,11 +20,29 @@ android {
 }
 
 dependencies {
+
     compile fileTree(dir: 'libs', include: ['*.jar'])
     androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
+
+    compile 'com.android.support:support-v4:26.+'
```

Comment:
Pozmieniaj wersje bibliotek na konkretne, bez plusów. Aktualnie najnowsze to 26.0.1. Żeby to zrobić musisz dopisać adres repozytorium do projektowego pliku `build.gradle`. Spójrz na komentarz do niego, tam szczegóły.

Reply:
Jak wpisuję wersję 26.0.1. to wyrzuca błędy i nie chce zsynchronizować gradle, wpisałem wersję 26.0.0-alpha1 i działa